app/helpers/checkouts_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Helper for checkouts views
+module CheckoutsHelper
+  def render_checkout_title(checkout)
+    title = checkout.title
+    title += " / #{checkout.author}" if checkout.author.present?
+    link_to title, "https://catalog.libraries.psu.edu/catalog/#{checkout.catkey}"
+  end
```

Comment:
I like this. I wonder if we should just use this in fines too just for consistency.

Reply:
I personally prefer the consistency as well like you. The current system has it like you had in fines with linking the word `Details`. I went on my own a bit here :p Let's see what Ruth will think about this way?

Reply:
Yep,I agree.